Samples/SimpleIslandApp/MainPage.xaml

```diff
@@ -0,0 +1,15 @@
+﻿<Page
+    x:Class="SimpleIslandApp.MainPage"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:local="using:SimpleIslandApp"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    mc:Ignorable="d">
+
+    <StackPanel>
+        <Button x:Name="Button" Click="ClickHandler">Click Me</Button>
+        <TextBox Text="Text goes here" />
+        <WebView2 Source="http://bing.com" Width="800" Height="600"/>
```

Comment:
>Width="800" Height="600" [](http://example.com/codeflow?start=43&length=24)

I assume this element is getting clipped currently when the island size isn't big enough? I suggest either changing the layout to size to available space, or add a ScrollViewer (though a WebView2 in a ScrollViewer isn't great).

Reply:
Great catch,!  I meant to change this.